src/app/pages/swap/components/swap-details/bitcoin-swap-details.tsx

```diff
@@ -0,0 +1,76 @@
+import { SwapSelectors } from '@tests/selectors/swap.selectors';
+import BigNumber from 'bignumber.js';
+import { useFormikContext } from 'formik';
+import { HStack, styled } from 'leather-styles/jsx';
+
+import { ChevronRightIcon } from '@leather.io/ui';
+import {
+  convertAmountToBaseUnit,
+  createMoney,
+  createMoneyFromDecimal,
+  formatMoneyPadded,
+  isUndefined,
+  satToBtc,
+} from '@leather.io/utils';
+
+import type { SwapFormValues } from '@shared/models/form.model';
+
+import { useSwapContext } from '@app/pages/swap/swap.context';
+
+import type { BitcoinSwapContext } from '../../providers/bitcoin-swap-provider';
+import { SwapDetailLayout } from './swap-detail.layout';
+import { SwapDetailsLayout } from './swap-details.layout';
+
+function RouteNames() {
+  return (
+    <HStack gap="space.01">
+      <styled.span>BTC</styled.span>
+      <ChevronRightIcon variant="small" />
+      <styled.span>sBTC</styled.span>
+    </HStack>
+  );
+}
+
+export function BitcoinSwapDetails() {
+  const { swapData } = useSwapContext<BitcoinSwapContext>();
+  const { values } = useFormikContext<SwapFormValues>();
+
+  if (isUndefined(values.swapAssetBase) || isUndefined(values.swapAssetQuote)) return null;
+
+  const maxSignerFee = satToBtc(swapData.maxSignerFee ?? 0);
+
+  const formattedMinToReceive = formatMoneyPadded(
+    createMoneyFromDecimal(
+      new BigNumber(values.swapAmountQuote).minus(maxSignerFee),
+      values.swapAssetQuote.balance.symbol,
+      values.swapAssetQuote.balance.decimals
+    )
+  );
```

Comment:
```suggestion
  const minToReceive = createMoneyFromDecimal(
    new BigNumber(values.swapAmountQuote).minus(maxSignerFee),
    values.swapAssetQuote.balance.symbol,
    values.swapAssetQuote.balance.decimals
  );
```

Added this [convention lately to notion](https://www.notion.so/trustmachines/Defer-data-formatting-160d74b694c78083b46dd1f7a892ad44?pvs=4)